src/components/molecules/EventDisplay/EventDisplay.tsx

```diff
@@ -6,9 +6,10 @@ import { Room } from "types/rooms";
 
 import { formatHourAndMinute, getCurrentTimeInUTCSeconds } from "utils/time";
 import { WithId } from "utils/id";
-
```

Comment:
This should be restored

Reply:
done

Reply:
I don't see what was mentioned here, but I guess it's done

Reply:
The very minor change was restoring the line break in line 9.